integration-cli/docker_api_containers_test.go

```diff
@@ -765,7 +765,7 @@ func (s *DockerAPISuite) TestContainerAPIPostCreateNull(c *testing.T) {
 		"OpenStdin":true,
 		"StdinOnce":true,
 		"Env":[],
-		"Cmd":"ls",
+		"Cmd":["ls"],
```

Comment:
We can probably remove this test entirely; looking at the related issue and the fix; this was because we were using our own custom JSON unmarshaling, which tried to be smart to detect types, which led to `null` being converted to a `"null"` string; see

- https://github.com/moby/moby/issues/7941
- https://github.com/moby/moby/pull/8273